doc/source/_static/myst-nb.css

```
@@ -0,0 +1,57 @@
+/* MyST-NB
+
+This stylesheet targets elements output by MyST-NB that represent notebook
+cells.
+
+In some cases these rules override MyST-NB. In some cases they override PyData
+Sphinx Theme or Sphinx. And in some cases they do not override existing styling
+but add new styling. */
+
+/*
+Undo the border that MyST-NB puts on the input cell
+*/
+div.cell div.cell_input {
+  border: none;
+}
+
+/*
+Undo border radius on code cells (this styling actually comes from Sphinx, not MyST-NB)
+*/
+div.cell pre,
+div.cell div.highlight,
+div.pywt-handcoded-cell-output,
+div.pywt-handcoded-cell-output pre,
+div.pywt-handcoded-cell-output div.highlight {
+  border-radius: 0;
+}
+
+/*
+Add border to output container, remove background color.
+*/
+div.cell div.cell_output,
+div.pywt-handcoded-cell-output {
+  background-color: var(--pst-color-background);
+  border-radius: 0;
+  border-color: var(--pst-color-border-muted);
+  border-style: solid;
+  border-width: 1px;
+  border-top-style: double;
+  border-top-width: medium;
+}
+div.cell div.cell_output div.output,
+div.cell div.cell_output pre,
+div.pywt-handcoded-cell-output pre {
+  background-color: transparent;
+  border: none;
+}
+
+/*
+Remove extra whitespace
+*/
+div.cell div.cell_output,
+div.cell div.cell_output > :first-child {
+  margin-top: 0;
+}
+div.pywt-handcoded-cell-output {
+  margin-top: -1em;
+}
```

```suggestion
div.pywt-handcoded-cell-output {
  margin-top: -1em;
}

```

nit: no whitespace at the end of the file

---

fixed! thank you